apps/www/content/docs/installation/gatsby.mdx

```diff
@@ -7,10 +7,10 @@ description: Install and configure Gatsby.
 
 ### Create project
 
-Start by creating a new Gatsby project using `Gatsby CLI`:
+Start by creating a new Gatsby project using `create-gatsby`:
 
 ```bash
-npx gatsby new
+npm init gatsby
```

Comment:
We use `npm init` / `yarn create` normally for our Quick Start guides, see https://www.gatsbyjs.com/docs/quick-start/